models/netsuite2/netsuite2__income_statement.sql

```diff
@@ -106,28 +107,31 @@ income_statement as (
         , transaction_details.{{ var('income_statement_transaction_detail_columns') | join (", transaction_details.")}}
 
         {% endif %}
-    
+
+        , -converted_amount_using_transaction_accounting_period as converted_amount
         
     from transactions_with_converted_amounts
 
     join transaction_lines as transaction_lines
         on transaction_lines.transaction_line_id = transactions_with_converted_amounts.transaction_line_id
             and transaction_lines.transaction_id = transactions_with_converted_amounts.transaction_id
 
-    left join classes 
-        on classes.class_id = transaction_lines.class_id
+    left join departments 
+        on departments.department_id = transaction_lines.department_id
+    
+    left join accounts 
+        on accounts.account_id = transactions_with_converted_amounts.account_id
 
     left join locations
-        on locations.location_id = transaction_lines.location_id
+        on accounts.location_id = transaction_lines.location_id
```

Comment:
why `accounts.location_id` instead of `locations.location_id`?

Reply:
Should not be. Just reverted.